extensions/amp-image-slider/0.1/amp-image-slider.js

```diff
@@ -153,76 +159,77 @@ export class AmpImageSlider extends AMP.BaseElement {
     this.registerAction('seekTo', invocation => {
       const {args} = invocation;
       if (args) {
-        let value;
-        if (args['value'] !== undefined) {
-          value = args['value'];
-          user().assertNumber(value,
+        if (args['percent'] !== undefined) {
+          const percent = args['percent'];
+          user().assertNumber(percent,
               'value to seek to must be a number');
-        } else if (args['percent'] !== undefined) {
-          value = args['percent'];
-          user().assertNumber(value,
-              'percent to seek to must be a number');
-          value *= 0.01; // percentage to 0-1 value
-        }
-        if (value !== undefined) {
-          this.updatePositions_(value);
+          this.updatePositions_(percent);
         }
       }
     }, ActionTrust.LOW);
 
+    const initialPercentString = this.element.getAttribute('initial-percent');
```

Comment:
Is the naming here finalized? If so we will need to validate this as well in your validation PR. Otherwise take it out.

Reply:
Held for P1